views/teams/topics.hbs

```diff
@@ -85,23 +85,23 @@
                         <div class="card-header">
                             <div class="input-group">
                                 <a class="btn btn-secondary btn-sm btn-hidden-toggle" href="{{../_id}}"
-                                    title="Thema wieder anzeigen">
-                                    <i class="fa fa-eye-slash" aria-hidden="true"></i>
+                                    title="{{$t "teams.topics.button.displayTopicAgain"}}">
+                                    <i class="fa fa-eye-slash"></i>
                                 </a>
                                 <p class="topic-label ml-1">{{../name}}</p>
                                 <div class="right-btn-group">
                                     <a class="delete-topic btn btn-secondary btn-secondary btn-sm"
                                         href="/teams/{{../courseId}}/topics/{{../_id}}" target="_blank"
                                         data-method="DELETE" data-name="{{../name}}" redirect="/teams/{{../courseId}}"
-                                        title="Thema löschen">
-                                        <i class="fa fa-trash-o" aria-hidden="true"></i>
+                                        title="{{$t "teams.topics.button.deleteTopic"}}">
+                                        <i class="fa fa-trash-o"></i>
                                     </a>
-                                    <button class="btn btn-secondary btn-sm move-handle" title="Thema verschieben">
```

Comment:
Why You are changing all buttons to a tag ?

Reply:
All changes happened because of merging. I havent done this intentionally! should have checked all the changes carefully.